PWGGA/GammaConv/AliAnalysisTaskConvCaloCalibration.cxx

```diff
@@ -903,6 +907,18 @@ void AliAnalysisTaskConvCaloCalibration::UserCreateOutputObjects(){
     if(temp) fOutputContainer->Add(temp->GetCaloTrackMatcherHistograms());
   }
 
+  // for electron selection
+  if(fUseEletronMatchingCalibration == 1){
+    fElecSelector=(AliDalitzElectronSelector*)AliAnalysisManager::GetAnalysisManager()->GetTask("ElectronSelector");
+    if(!fElecSelector){printf("Error: No ElectronSelector");return;} // GetV0Reader
```

Comment:
This should better be an AliFatal. Otherwise you might have unintended behavior in the code.

Reply:
Okay, I changed it. Thanks!